README.md

```diff
@@ -6,8 +6,9 @@ FireCloud Orchestration Service
 * Github Repository: [https://github.com/broadinstitute/firecloud-orchestration/](https://github.com/broadinstitute/firecloud-orchestration/)
 
 ## IntelliJ Setup
-* Configure the SBT plugin
-* Open this source directory with File -> Open
+* IntelliJ IDEA can be downloaded here : https://www.jetbrains.com/idea/ . It is an 'Intelligent Java IDE'
+* Configure the SBT plugin.  The SBT plugin for IntelliJ IDEA can be downloaded here : https://plugins.jetbrains.com/
```

Comment:
Well, first of all, I am using IntelliJ for the first time.  That said...., I have been able to find a "Configure Plugin" box for IntelliJ but haven't found a "Download and Install Plugins" box inside it.  That's probably why I referenced the URL.  I did use that jetbrain link to download a zip file of the sbt plugin.....